<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserEditor</class>
 <widget class="QWidget" name="CUserEditor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User Editor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>First Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="firstNameLineEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Phone Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="phoneNumberLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Last Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lastNameLineEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Email Address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="emailAddressLineEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="addressLineEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Country:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="countryLineEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>City:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="cityLineEdit"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Province/State:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="provinceStateLineEdit"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Postal/Zip:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="postalZipLineEdit"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Emergency Contact Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="emergencyContactNameLineEdit"/>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Tag Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="tagIdLineEdit"/>
   </item>
   <item row="6" column="5" colspan="2">
    <widget class="QPushButton" name="readPushButton">
     <property name="text">
      <string>Read Tag</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Emergency Contact Phone:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="emergencyContactPhoneLineEdit"/>
   </item>
   <item row="7" column="5" colspan="2">
    <widget class="QPushButton" name="generatePushButton">
     <property name="text">
      <string>Generate Tag</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Track Registration Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="trackRegistrationLineEdit"/>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QPushButton" name="writePushButton">
     <property name="text">
      <string>Write Tag</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>CA Registration:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="cARegistrationLineEdit"/>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QCheckBox" name="wantsEmailReportsCheckBox">
     <property name="text">
      <string>Wants Email Reports</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>664</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="5">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
